gokart/run.py

```diff
@@ -96,7 +98,14 @@ def _try_to_send_event_summary_to_slack(slack_api: Optional[gokart.slack.SlackAP
     slack_api.send_snippet(comment=comment, title='event.txt', content=content)
 
 
-def run(cmdline_args=None, set_retcode=True):
+def _dump_task_info(cmdline_args: List[str], dump_task_info_path: Optional[str]):
+    if dump_task_info_path is not None:
+        result, unique_id = get_task_info(cmdline_args=cmdline_args)
+        task_info_target = make_target(file_path=dump_task_info_path, unique_id=unique_id, processor=None, redis_params=None)
+        task_info_target.dump(obj=result, lock_at_dump=False)
+
+
+def run(cmdline_args=None, set_retcode=True, dump_task_info_path: Optional[str] = None):
     cmdline_args = cmdline_args or sys.argv[1:]
```

Comment:
[IMO] You can get `dump_task_info_path` from `cmdline_args` so that we can use `dump_task_info_path` as usual way as other luigi options